src/types.ts

```diff
@@ -460,7 +460,8 @@ type ZodStringCheck =
   | { kind: "startsWith"; value: string; message?: string }
   | { kind: "endsWith"; value: string; message?: string }
   | { kind: "regex"; regex: RegExp; message?: string }
-  | { kind: "trim"; message?: string };
+  | { kind: "trim"; message?: string }
+  | { kind: "utc"; options?: { milliseconds?: Boolean }; message?: string };
```

Comment:
Went with an options object here for better visual clarity.

`.utc(true)`/`.utc(false)` did not read as well to me.